### DISCUSSION TITLE
Request: Support for 3Dc / BC5 / ATI2 format DDS files

### DISCUSSION THREAD

**Discussion by kanasc** [ImageMagick/ImageMagick#4728] [Development]
Would it be possible to add support for 3Dc / BC5 / ATI2 format DDS files? It's most often used for normal maps (and sometimes speculars) because it only contains two 8-bit channels. I believe most editors that support them simply assign them to red and green, and create a blue channel that's solid white or black.

3Dc+ / BC4 / ATI1 is similar but only has one channel. Some editors handle it by treating it as single-channel greyscale, some by duplicating it in each of red/green/blue.

MSDN articles on [BC4](https://docs.microsoft.com/en-gb/windows/win32/direct3d10/d3d10-graphics-programming-guide-resources-block-compression#bc4) and [BC5](https://docs.microsoft.com/en-gb/windows/win32/direct3d10/d3d10-graphics-programming-guide-resources-block-compression#bc5)

**Comment by jonri** [comment]
I'd like to see this as well.  On the surface it doesn't seem like BC4/5 would be very hard to do, since the compression is the same as the alpha channel for DXT5 which is already supported.  The other half would just be getting the file headers right.  I'm not an expert in the file format, but I might take a stab at this sometime.

**Comment by NXTler** [comment]
So is or will there be supports for those formats?

**Comment by dlemstra** [comment]
Looks like someone change this into an issue #5919.
